packages/plugins/rubygem-foreman_openscap/rubygem-foreman_openscap.spec

```diff
@@ -36,10 +36,8 @@ BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.4.1
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 9.0.0
-BuildRequires: %{?scl_prefix}npm(graphql-tag) >= 2.11.0
-BuildRequires: %{?scl_prefix}npm(graphql-tag) < 3.0.0
-BuildRequires: %{?scl_prefix}npm(graphql) >= 15.5.0
-BuildRequires: %{?scl_prefix}npm(graphql) < 16.0.0
```

Comment:
For now let's keep `graphql-tag` requirement here as it's not in SCAP package.json or foreman-js

`graphql` is already part of Foreman-js: https://github.com/theforeman/foreman-js/blob/master/packages/vendor-core/package.json#L51 so it should work without it 🤞🏼

Reply:
Did that